sdk/appconfiguration/src/configuration/mod.rs

```diff
@@ -0,0 +1,77 @@
+use azure_core::auth::TokenCredential;
+use futures::{executor::block_on, stream::StreamExt};
+use std::{collections::HashMap, sync::Arc};
+
+#[derive(Clone)]
+pub struct ConfigurationExplorer {
+    client: azure_svc_appconfiguration::Client,
+}
+
+impl ConfigurationExplorer {
+    pub fn new(token_credential: Arc<dyn TokenCredential>) -> Self {
+        let name =
+            std::env::var("AZCONFIG_NAME").expect("Missing AZCONFIG_NAME environment variable.");
+        let endpoint = format!("https://{name}.azconfig.io");
+        let scopes = &["https://azconfig.io"];
+
+        let client = azure_svc_appconfiguration::Client::builder(token_credential)
+            .endpoint(endpoint)
+            .scopes(scopes)
+            .build();
+
+        Self { client }
+    }
+
+    pub async fn get_value_async(&self, key: String) -> String {
+        let value = match self.client.clone().get_key_value(key).await {
+            Ok(rs) => rs.value,
+            Err(err) => {
+                log::debug!("*ERROR :  {:?}", err);
+                None
+            }
+        };
+
+        match value {
+            Some(value) => value,
+            None => String::from(""),
+        }
+    }
+
+    pub fn get_value(&self, key: String) -> String {
+        block_on(self.get_value_async(key))
+    }
+
+    pub async fn get_values_async(&self, label: String) -> HashMap<String, String> {
```

Comment:
As the underlying call to azure_svc_appconfiguration's `get_key_values().label()` takes an `Into<String>`, this should as well.  This way we don't need to require the caller to pass an owned value.

Reply:
agree, updated according you recommendation.